test/files/run/t4950.check

```diff
@@ -0,0 +1,13 @@
+Expected 7 lines, got 11
+--- expected
++++ actual
+@@ -2,2 +2,4 @@
+ scala> val 1 = 2
++           ^
++       warning: Pattern definition introduces Unit-valued member of 
+ scala.MatchError: 2 (of class java.lang.Integer)
+@@ -5,2 +7,4 @@
+ scala> val List(1) = List(1)
++               ^
++       warning: Pattern definition introduces Unit-valued member of 
+ 
```

Comment:
This doesn't look right.

Reply:
That's why it's worth warning. Possibly the way forward is not to emit the temp val.

Reply:
I meant the fact that it just ends with "of ", which actually now I guess is because it's scrubbing the REPL wrappers from the output.

Reply:
Ha, yes.

Reply:
Maybe shove this one is a `:paste -raw`?  Or... something?

My concern is trailing spaces risk being trimmed out by editors and causing spurious partest checkfile diffs.  It's also just a touch confusing.